### ISSUE TITLE
ExprAdapter is not defined

### ISSUE THREAD

**Issue by WenluZhang1** [claroty/enip-stack-detector#2]
Hi,

I tried to use the stack detector to test my device while I got the below error says `"NameError: name 'ExprAdapter' is not defined"`. I used python 2.7.17 and I have the construct 2.9.45 installed as shown in the graph. Would you please suggest what could be the possible reason for the error?

<img width="461" alt="notdefined" src="https://user-images.githubusercontent.com/49553326/210723392-119b4ce5-7488-4aae-bf9f-61cf97d961fd.png">

**Comment by SharonBrizinov** [comment]
Can you run this?
```python2 -V ; python2 -c "import construct; print(construct.__version__)"```

This is what I have - 
![Screen Shot 2023-01-05 at 11 47 22](https://user-images.githubusercontent.com/519424/210750525-536cd991-b91b-401e-9b44-19bafab966a9.png)

**Comment by WenluZhang1** [comment]
Thanks for your quick response. I tried the command and the result is shown below. Python2 doesn't print the construct version so I use pip show instead. 
<img width="452" alt="Picture2" src="https://user-images.githubusercontent.com/49553326/210945165-eda90bda-29fa-4455-9bb9-07efc828426d.png">
When I tried to install construct 2.9.45, the command `pip2 -m install construct==2.9.45 --user` returns error `no such option: -m`, so I use `pip2 -m install construct==2.9.45 --user` instead. Could that be the reason cause the error?
